code/modules/mob/living/simple_mob/subtypes/animal/giant_spider/broodmother.dm

```diff
@@ -31,7 +31,6 @@
 		/mob/living/simple_mob/animal/giant_spider/electric/broodling,
 		/mob/living/simple_mob/animal/giant_spider/hunter/broodling,
 		/mob/living/simple_mob/animal/giant_spider/lurker/broodling,
-		/mob/living/simple_mob/animal/giant_spider/nurse/broodling,
```

Comment:
Broodmother is itself adminspawn only, and it seems excessive to nerf its sub-spawns there